core/src/main/scala/org/apache/spark/h2o/converters/H2ORDDLike.scala

```diff
@@ -23,7 +23,8 @@ import water.fvec.{Frame, FrameUtils}
 /**
  * Contains functions that are shared between all H2ORDD types (i.e., Scala, Java)
  */
-private[converters] trait H2ORDDLike[T <: Frame] {
```

Comment:
That's while H2OContext is still in h2o. Once we move it over, the trait will be again package-private.